src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+        <h4>Payload First Frame</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte #</th>
+                    <th>Byte Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0</td>
+                    <td>Frame Counter (0x00)</td>
+                </tr>
+                <tr>
+                    <td>1</td>
+                    <td>Log Type</td>
+                </tr>
+                <tr>
+                    <td>2-3</td>
+                    <td>Data Length</td>
+                </tr>
+                <tr>
+                    <td>4-7</td>
+                    <td>Data</td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Payload Subsequent Frames</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte #</th>
+                    <th>Byte Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0</td>
+                    <td>Frame Counter</td>
+                </tr>
+                <tr>
+                    <td>1-7</td>
+                    <td>Data</td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Log Types</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>Parameter Type</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0</td>
+                    <td>Third Party Free Format Data</td>
+                </tr>
+                <tr>
+                    <td>1</td>
+                    <td>Reserved</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>Bluetooth Record</td>
+                </tr>
+                <tr>
+                    <td>3-9</td>
+                    <td>Reserved</td>
+                </tr>
+                <tr>
+                    <td>10</td>
+                    <td>Driver ID</td>
+                </tr>
+                <tr>
+                    <td>11</td>
+                    <td>Curve Logging</td>
+                </tr>
+                <tr>
+                    <td>12</td>
+                    <td>Logging With Timestamp</td>
+                </tr>
+                <tr>
+                    <td>13</td>
+                    <td>Protobuf Data</td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Type 0 Third Party Free Format Data</h4>
+        <p>The maximum size is 27 bytes. Rate limit is 500 logs per 10 minutes. If you exceed the rate limit, the GO device will stop taking data from the IOX.</p>
+        <table>
```

Comment:
we should have tables embedded in a div with the .table-container class until we come up with the final styling so it can be applied later on